src/course_bot/talk.clj

```diff
@@ -43,7 +43,7 @@
       (keyword (first args))
       nil)))
 
-(defn id-from-arg [tx text]
+(defn id-from-arg [text]
```

Comment:
If I don't mistake, we don't use this function. Remove it.

Reply:
Yeah, it is unused